test/over_react/util/react_wrappers_test.dart

```diff
@@ -1147,6 +1272,16 @@ class TestComponent extends react.Component {
   render() => Dom.div()();
 }
 
+/// Helper component for testing a Dart (react-dart) React component (version
+/// 2) with cloneElement.
+final TestComponent2Factory = react.registerComponent(() => new TestComponent2
+  ());
```

Comment:
This should move up to the previous line.  Function and constructor invocations _(or at a minimum - the opening `(` of one)_... stay with the function name / class name:

```dart
// Typical - when line length is reasonable
someFunction(arg1, arg2);
var foo = new Foo(arg1, arg2);

// When line length is exceeded (note that the continuation indent is after the opening (
someFunction(
    arg1, arg2);
var foo = new Foo(
    arg1, arg2);
```

Also note that continuation indents - when used - should be 4 spaces, not 2.  This should all be configurable in your IDE settings as well.

Reply:
👍 Thank you for the detailed explanation / example - it helped a lot to see the implementation